### PR TITLE
MEN-4395: Bump version to 2021-01-11

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>
(cherry picked from commit 560ec3a1b4afbbbac9b268eb366b309da6b7c56c)

### CODE REVIEW

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md [hunk 1]
```diff
@@ -34,8 +34,10 @@ Download the Raspberry Pi OS image with Mender integrated:
   * Download link: [Raspberry Pi 4 Model B][raspios-buster-lite-raspberrypi4-mender.img.xz]
 
 <!--AUTOVERSION: "mender-%.img.xz"/mender-convert-client -->
-[raspios-buster-lite-raspberrypi3-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2020-05-27-raspios-buster-lite-armhf/arm/2020-05-27-raspios-buster-lite-armhf-raspberrypi3-mender-2.4.2.img.xz
-[raspios-buster-lite-raspberrypi4-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2020-05-27-raspios-buster-lite-armhf/arm/2020-05-27-raspios-buster-lite-armhf-raspberrypi4-mender-2.4.2.img.xz
+[raspios-buster-lite-raspberrypi3-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2021-01-11-raspios-buster-armhf-lite/arm/2021-01-11-raspios-buster-armhf-lite-raspberrypi3-mender-latest.img.xz
```
Comment (lluiscampos) [inline 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:37]: Careful with the client version!

**Review by lluiscampos — COMMENTED** [review]
To master maybe (see https://github.com/mendersoftware/mender-docs/pull/1367#pullrequestreview-608631619), but to release branches we cannot update them this way.

On top of that, we shall not touch the mender (client) version in these branches.

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md (hunk 1 above)
Comment (oleorhagen) [reply 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:37]: Oh, didn't notice. They're versioned even. I guess the images don't exist for `2.4.2` then. Should I do a manual release of them now, anyhow?

Comment (lluiscampos) [reply 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:37]: As I said in the other PR, I don't think so.

**Pull request closed without merging**